util/newArchStatus.ts

```diff
@@ -6,22 +6,52 @@ export enum NewArchSupportStatus {
   Untested = 'untested',
 }
 
-export function getNewArchSupportStatus({ newArchitecture, github, expoGo }: Library) {
+//new function  to fix new architecture support detection
+export function getNewArchSupportStatus({
+  newArchitecture,
+  github,
+  expoGo,
+  lastPublish,
+  maintained,
+  reactNativeVersion,
```

Comment:
These does not exist, you can see what information is available by looking at `Library` type.

![Screenshot 2025-03-10 at 11 12 51](https://github.com/user-attachments/assets/48e1e888-4993-4078-b492-0f6666c53da1)